libraries/lib-ffmpeg-support/impl/avcodec/60/AVCodecIDLookup.cpp

```diff
@@ -0,0 +1,37 @@
+extern "C"
+{
+#include "../../avutil/58/avconfig.h"
```

Comment:
60 using 58 again, comment ?

Reply:
This files are literally copied, none of the original files have a comment about different versions between avutil and the rest of the libraries. 

The difference between versions is due to the difference inside the FFmpeg itself.